doc/requirements.txt

```
@@ -26,7 +26,7 @@ tensorflow==2.11.1; platform_machine == "x86_64"
 tensorflow_macos==2.9.0; sys_platform == "darwin" and platform_machine == "arm64"
 tensornetwork==0.3
 toml
-torch==1.9.0+cpu; sys_platform != "darwin" and python_version < "3.10"
+torch==1.9.0; sys_platform != "darwin" and python_version < "3.10"
 torch==1.9.0; sys_platform == "darwin" and python_version < "3.10"
```

These lines are now redundant and could be merged into
```suggestion
torch==1.9.0; python_version < "3.10"
```

---

And can be removed all together when we stop support for Python 3.9 :tada: